pymc3/distributions/continuous.py

```diff
@@ -1297,20 +1300,32 @@ def logcdf(self, value):
         Parameters
         ----------
         value: numeric
-            Value(s) for which log CDF is calculated. If the log CDF for multiple
-            values are desired the values must be provided in a numpy array or theano tensor.
+            Value(s) for which log CDF is calculated.
 
         Returns
         -------
         TensorVariable
         """
-        value = floatX(tt.as_tensor(value))
-        a = floatX(tt.as_tensor(self.alpha))
-        b = floatX(tt.as_tensor(self.beta))
-        return tt.switch(
-            tt.le(value, 0),
-            -np.inf,
-            tt.switch(tt.ge(value, 1), 0, tt.log(incomplete_beta(a, b, value))),
+        # incomplete_beta function can only handle scalar values (see #4342)
+        if np.ndim(value):
+            raise TypeError(
+                "Beta.logcdf expects a scalar value but received a {}-dimensional object.".format(
+                    np.ndim(value)
```

Comment:
Any reason you're not using f-strings here?

Reply:
I was not sure if the oldest supported version of pymc3 already includes f-strings. If it does I would be happy to change it, as I also prefer them.

Reply:
Updated it to use f-strings